src/kbucket/bucket.rs

```diff
@@ -938,11 +939,13 @@ pub mod tests {
                 }
                 Action::UpdateStatus(pos, status) => {
                     if let Some(key) = self.key_of_pos(pos) {
+                        // TODO: action result ignored
```

Comment:
You've fixed most the others, these too?

Reply:
leaving this up since the return type has a `must_use` and I really don't know how this should be treated